index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/722afd38e8.js" crossorigin="anonymous"></script>
+</head>
+<body>
+   
+  <section class="header">
+    <div class="my-logo"><span>My Logo</span></div>
+
+    <div class="dropdown">
+        <button class="dropbtn"><img src="assets/images/button.png" alt=""></button>
+        <div class="dropdown-content">
+          <a href="#">Portfolio</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </div>
+      </div>
+
+  </section>
+   
+    <section class="headline">
+      
+        <h2>I’m Ahmed Glad to see you!</h2>
+        
+        <p>I’m a software developer! I can help you build a product,
+             feature or website Look through some of my work and experience! 
+             If you like what you see and have a project you need coded,
+             don’t hestiate to contact me.</p>  
+
+             <div class="Social"> 
```

Comment:
:red_circle: Kindly make sure you reduce the vertical white space between thee two blocs
|Your Design|Figma Design|
|:----------------:|:----------------:|
|![Screenshot from 2023-01-18 00-07-50](https://user-images.githubusercontent.com/26736582/213024231-0d265aea-f9ac-45c3-97f6-7104d046ec3c.png)|![Screenshot from 2023-01-18 00-08-05](https://user-images.githubusercontent.com/26736582/213024286-0dbb2e8f-8c65-4e88-bb28-29a4ee4a5446.png)|

Reply:
resolved :+1: